.github/workflows/frontend.yml

```diff
@@ -0,0 +1,93 @@
+name: Frontend
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release**'
+      - 'feature**'
+      - 'fix**'
+      - 'ci**'
+    tags:
+      - '**'
+    paths:
+    - 'frontend/**'
+    - 'enterprise/frontend/**'
+    - 'docs/**'
+    - '**/yarn.lock'
+    - '.github/workflows/**'
+
+jobs:
+
+  fe-linter-flow:
+    runs-on: ubuntu-20.04
```

Comment:
Is there a reason we're targeting this specific version of Ubuntu instead of `ubuntu-latest`? We want to have controlled environment?

Reply:
Correct. Actually using `ubuntu-latest` gives the following warning:

> Ubuntu-latest workflows will use Ubuntu-20.04 soon. For more details, see https://github.com/actions/virtual-environments/issues/1816